src/libraries/Common/tests/Tests/System/StringTests.cs

```diff
@@ -3776,12 +3776,7 @@ public static void IsWhiteSpaceTrueLatin1()
             Random rand = new Random(42);
             for (int length = 0; length < 32; length++)
             {
-                char[] a = new char[length];
-                for (int i = 0; i < length; i++)
-                {
-                    a[i] = s_whiteSpaceCharacters[rand.Next(0, s_whiteSpaceCharacters.Length - 1)];
-                }
-
+                char[] a = rand.GetItems<char>(s_whiteSpaceCharacters, length);
```

Comment:
One of the reasons I like to see APIs used in the same PR they're added is to help understand if the design is correct.

What was the reason for choosing to add the GetString APIs to RandomNumberGenerator but not to Random?

Reply:
@tannergooding "I'd like to take both the `Shuffle` and `GetItems` for `Random` [...]. The `GetString` ones I think might require more discussion and depth. And at the very least people can fill the need with the other APIs."

"What's the reason `GetString` more controversial?"

"There is a clear use case for specializing `RandomNumberGenerator`. It wasn't clear if there is one for `Random`. We can always take that as a separate review item".

https://youtu.be/73BAn9B3ni4?t=9187

Reply:
Ok, well, at least for testing, this PR shows a clear use case for it :) A bunch of the tests are creating a new char[] simply to feed that into string's ctor.